keyboards/kin80/readme.md

```diff
@@ -0,0 +1,15 @@
+# Kin80
+
+![Kin80 controller](https://i.imgur.com/Im19DdH.jpg)
+
+Kin80 is a set of replacement PCBs and switch mounting plates for [Kinesis Contoured keyboards](https://deskthority.net/wiki/Kinesis_Contoured).
+
+* Keyboard Maintainer: [Dmitriy Nosachev](mailto:quartz64@gmail.com)
+* Hardware Supported: Kin80 PCB
+* Hardware Availability: [Open Source (CC-BY-SA](https://github.com/DmNosachev/kinesis80)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
Done.